onnxruntime/core/graph/model_editor_api_types.h

```diff
@@ -173,6 +173,8 @@ struct ModelEditorGraph : public OrtGraph {
 
   const std::string& GetName() const override { return name; }
 
+  const ORTCHAR_T* GetModelPath() const override { return ORT_TSTR(""); }
```

Comment:
>return ORT_TSTR(""); [](http://example.com/codeflow?start=51&length=20)

Can the model path be part of the base class so this would be a meaningful impl?

Reply:
Would a model from model editor every have a valid model path?

Reply:
Updated